Remove experimental flag for redirects config (#7686)
withastro/astro: .changeset/dry-beers-grow.md

---
'astro': minor
---

Redirects configuration

This change moves the `redirects` configuration out of experimental. If you were previously using experimental redirects, remove the following experimental flag:

```js
experimental: {
  redirects: true,

> }

Reviewer: ``````suggestion
}
```

If you have been waiting for stabilization before using redirects, now you can do so. Check out [the docs on redirects](https://docs.astro.build/en/core-concepts/routing/#redirects) to learn how to use this built-in feature.
``````
UGH.. nested backticks *almost* won again! *<shakes fist>*

This corrects the commit of a wonkily-formatted suggestion, that cut off some of the intended code.

---
'astro': minor
---

Redirects configuration

> This change moves the `redirects` configuration out of experimental, meaning if you are currently using the:
> 
> ```js
> experimental: {
>   redirects: true,
> }
> ```
> 
> You can simply remove this flag. If you have been waiting on using redirects before it stabilized, how you can do so. Checkout [the docs](https://docs.astro.build/en/core-concepts/routing/#redirects) to learn how to use the feature.

Reviewer: ```suggestion
This change moves the `redirects` configuration out of experimental. If you were previously using experimental redirects, remove the following experimental flag:

```js
experimental: {
  redirects: true,
}
```

If you have been waiting for stabilization before using redirects, now you can do so. Check out [the docs on redirects](https://docs.astro.build/en/core-concepts/routing/#redirects) to learn how to use this built-in feature.
```

Just tightened up some wording, and made it a little more forceful re: what to do now!
